### PR TITLE
Modified arrayutils function

### PR DESCRIPTION
**Pull request by ignaciosantise**
### Summary
Modified `stringArrayToObject` method to work as expected and to pass the tests

#### Test failure
fail

#### Test success
success

### CODE REVIEW

File: generators/app/templates/src/utils/arrayUtils.js [hunk 1]
```diff
@@ -9,5 +9,5 @@ export function stringArrayToObject(actionsArray, namespace = '') {
   if (actionsArray.some(actionName => !actionName || typeof actionName !== 'string')) {
     throw new Error('Action names must be strings and must not be empty');
   }
-  return Immutable(actionsArray).asObject(actionName => [actionName, `${namespace}:${actionName}`]);
+  return Immutable(actionsArray).asObject(actionName => [actionName, `${namespace}${actionName}`]);
```
Comment (sbalay) [inline generators/app/templates/src/utils/arrayUtils.js]: I'd keep the colon for readability, any particular reason for removing it?

Comment (sbalay) [reply generators/app/templates/src/utils/arrayUtils.js]: We can conditionally remove it if there is no namespace

Comment (mvbattan) [inline generators/app/templates/src/utils/arrayUtils.js]: Shouldn't this be
```
namespace ? `${namespace}/${actionName}` : `${actionName}`
```
?

Comment (guidoprinc) [reply generators/app/templates/src/utils/arrayUtils.js]: In sura we changed it to 
```
return Immutable(actionsArray).asObject(actionName => [
    actionName,
    `${namespace}${namespace ? ':' : ''}${actionName}`
  ]);
```

Comment (ignaciosantise) [reply generators/app/templates/src/utils/arrayUtils.js]: you are right! removing that colon was not a good decision

Comment (ignaciosantise) [reply generators/app/templates/src/utils/arrayUtils.js]: we could use the slash to separate the namespace and the action name! is `namespace/actionName` clearer than `namespace:actionName`?

@sbalay @mvbattan @guidoprinc

Comment (sbalay) [reply generators/app/templates/src/utils/arrayUtils.js]: We are also using this for i18n declarations. I think that the `colon` is clearer but what you guys prefer is better.

Comment (mvbattan) [reply generators/app/templates/src/utils/arrayUtils.js]: IMHO using a slash will keep consistency across action names (e.g. those coming from redux form).
On the other hand, a colon will help to distinguish between our actions and other libraries actions.

**Pull request merged**